pkg/server/exec/handlers.go

```diff
@@ -0,0 +1,49 @@
+package exec
+
+import (
+	"github.com/luizalabs/teresa/pkg/goutil"
+	execpb "github.com/luizalabs/teresa/pkg/protobuf/exec"
+	"github.com/luizalabs/teresa/pkg/server/database"
+	"google.golang.org/grpc"
+)
```

Comment:
Maybe skip a line to separate the google import

Reply:
I'm using`goimports` 🤔